recipes/msmtools/meta.yaml

```diff
@@ -38,8 +35,10 @@ test:
 about:
   home: http://github.com/markovmodel/msmtools
   license: LGPL-3 
-  license_file: LICENSE.txt
```

Comment:
You still want to include this line though.  That's how conda-build knows what the license file is.  Please re-include this